migrations/Version202209131056091465_qtiItemPci.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\qtiItemPci\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use Doctrine\Migrations\Exception\IrreversibleMigration;
+use oat\qtiItemPci\scripts\install\RegisterPciMathEntry;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version202209131056091465_qtiItemPci extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Register new version of Math Entry PCI with save correct response feature';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addReport(
+            $this->propagate(
+                new RegisterPciMathEntry()
+            )(
```

Comment:
are you sure this syntax is correct

Reply:
It is the same way as on the previous migration